presidio-anonymizer/presidio_anonymizer/entities/engine_request.py

```diff
@@ -0,0 +1,98 @@
+"""
+Engine request entity.
+
+It get the data and validate it before the engine receives it.
+"""
+import logging
+
+from presidio_anonymizer.anonymizers.fpe import FPE
+from presidio_anonymizer.anonymizers.hash import Hash
+from presidio_anonymizer.anonymizers.mask import Mask
+from presidio_anonymizer.anonymizers.redact import Redact
+from presidio_anonymizer.anonymizers.replace import Replace
+from presidio_anonymizer.entities.analyzer_result import AnalyzerResult
+from presidio_anonymizer.entities.analyzer_results import AnalyzerResults
+from presidio_anonymizer.entities.invalid_exception import InvalidParamException
+
+
+class AnonymizerEngineRequest:
+    """The request the engine will get and act on."""
```

Comment:
```suggestion
    """Input validation for the anonymize process"""
```
Isn't it more accurate about what is does?

Reply:
DONE.